charts/datadog/values.yaml

```diff
@@ -503,10 +503,10 @@ datadog:
     enableOOMKill: false
 
     # datadog.systemProbe.enableRuntimeCompiler -- Enable the runtime compiler for eBPF probes
-    enableRuntimeCompiler: false
+    # enableRuntimeCompiler: false
```

Comment:
I think you can do this:
```suggestion
    enableRuntimeCompiler:
```

Reply:
But that might cause a change in your other template code. The benefit of having it defined, but unset, is that it shows up in documentation.

Reply:
Maybe I'm wrong about the documentation though, I'm not 100% sure how that is processed.

Reply:
Unfortunately this doesn't work for our use case - see my note in the README.